Automattic/woocommerce-payments
# New Tracks event on KYC modal open

Fixes #4980 

#### Changes proposed in this Pull Request

- Add a new Tracks event when the new informational KYC modal is opened.
- This can be used in funnels to track if it improves conversion/completion of KYC.









#### Testing instructions





* Checkout this branch and run `npm run build` to create a `.zip`.
* Spin up a new JN site with the WooCommerce plugin installed.
* Upload and activate the WCPay plugin by going to Plugins > Add New and clicking `Upload`.
* Go to `WooCommerce > Settings > Advanced > WooCommerce.com` and  check the `Allow usage to be tracked` checkbox and save the change.
* Ensure you don't have adblocked or anything similar enabled.
* Go to `Payments` and open the KYC modal by clicking on the `Learn more about how to receive payouts.` text shown in the image below.

<img width="799" alt="Screenshot 2022-10-27 at 16 30 57" src="https://user-images.githubusercontent.com/11770181/198333273-2626eed0-ef0a-4a2a-b814-dfdbb4c634e6.png">

* Wait around 20 minutes (a good chance for a coffee 😂), then go to Tracks Live View (you can get there by going to MC and finding it in the `Metrics` section on the top header). You should be able to see a request to `pixel.wp.com` appear in the network tab of your browser console after clicking to open the modal.
* Search for the event name: `%wcpay_connect_account_kyc%`
* You should see a matching event with the full event name: `wcadmin_wcpay_connect_account_kyc_modal_opened`

-------------------


**Post merge**

* Validate the event as shown [here](https://github.com/Automattic/tracks-events-validation-status#event-status-labels)
* Get approval from the data team to ensure this method of tracking the event matches their expectations.
